node/node.go

```diff
@@ -384,7 +386,11 @@ func (nc *Config) Build(ctx context.Context) (*Node, error) {
 	//nwork := bsnet.NewFromIpfsHost(innerHost, router)
 	bswap := bitswap.New(ctx, nwork, bs)
 	bservice := bserv.New(bs, bswap)
-	fetcher := net.NewBitswapFetcher(ctx, bservice, blkValid)
+	//fetcher := net.NewBitswapFetcher(ctx, bservice, blkValid)
+
+	bridge := ipldbridge.NewIPLDBridge()
+	gsn := gsnet.NewFromLibp2pHost(peerHost)
+	fetcher := net.NewGraphSyncFetcher(ctx, gsn, bridge, bs, blkValid)
```

Comment:
So that we can keep PRs small and make rapid progress on the path here, I suggest not checking these changes in yet, which means we can check in code above before it's necessarily full and complete.